diff --git a/PluginManager.cpp b/PluginManager.cpp
index 4e52eb50..3708f470 100644
--- a/PluginManager.cpp
+++ b/PluginManager.cpp
@@ -109,7 +109,7 @@ void PluginManager::ScanAndLoadPluginsFrom(const filesystem::path & plugins_dir,
     | Get a list of all files in the plugins directory          |
     \*---------------------------------------------------------*/
 
-    for(const filesystem::directory_entry& entry: filesystem::directory_iterator(plugins_dir))
+    for(const filesystem::directory_entry& entry: filesystem::recursive_directory_iterator(plugins_dir))
     {
         if(filesystem::is_directory(entry.path()))
         {
